ractor_cluster/src/node/client.rs

```diff
@@ -51,6 +54,76 @@ impl From<MessagingErr> for ClientConnectErr {
     }
 }
 
+/// TODO Either move this into ClientConnectErr or implement `std::error::Error::cause`
+#[derive(Debug)]
```

Comment:
Can't you just derive Error? rather than implement it specifically?

Reply:
I don't think so, at least not without using external libraries such as [anyhow](https://docs.rs/anyhow/latest/anyhow/), but I didn't want to change your dependencies here without your input